bundle/bundle.go

```diff
@@ -128,6 +138,15 @@ type Location struct {
 	EnvironmentVariable string `json:"env,omitempty" yaml:"env,omitempty"`
 }
 
+// Validate the Location
+func (l Location) Validate() error {
+	forbiddenPath := "/cnab/app/outputs"
+	if strings.HasPrefix(l.Path, forbiddenPath) {
+		return fmt.Errorf("Path must not be a subpath of %q", forbiddenPath)
```

Comment:
Maybe add the actual `path` to the error message? And I think error message shouldn't start with an upper case, as they can be wrapped.

Reply:
Good call; added.